pkgs/applications/audio/musikcube/default.nix

```diff
@@ -54,9 +59,14 @@ stdenv.mkDerivation rec {
     libev
     libmicrohttpd
     ncurses
-    pulseaudio
     taglib
-  ] ++ lib.optional systemdSupport systemd;
+  ] ++ lib.optional systemdSupport [
```

Comment:
These should be lib.optionals

Reply:
D'oh! 🤦 Thanks @Artturin! I've put #172990 up for review to address this.
I do see uses of `lib.optional` is several other `default.nix` files, I should revisit the Nix documentation to understand the difference.